src/pymorize/caching.py

```diff
@@ -9,5 +12,33 @@
 
 def manual_checkpoint(data, rule):
     """Manually insert a checkpoint in the flow"""
-    logger.info(f"Manually inserting checkpoint")
-    return Completed(message="Checkoint reached", return_value=data)
+    logger.info("Manually inserting checkpoint")
+    return Completed(message="Checkpoint reached", return_value=data)
```

Comment:
This part just fixed a minor typo (`"Checkoint reached"` --> `"Checkpoint reached"`